playbooks/common/openshift-cluster/upgrades/pre/verify_health_checks.yml

```diff
@@ -11,3 +11,6 @@
       checks:
       - disk_availability
       - memory_availability
+      - docker_image_availability
+      - etcd_imagedata_size
+      - etcd_volume
```

Comment:
Similarly, this might be not-so-good. If an upgrade is actually necessary to reduce etcd usage, this check prior to upgrade could block getting the cluster to a healthier state.